modules/core/04-channel/v2/keeper/keeper.go

```diff
@@ -180,38 +180,38 @@ func (k *Keeper) SetNextSequenceSend(ctx context.Context, sourceID string, seque
 
 // AliasV1Channel returns a version 2 channel for the given port and channel ID
 // by converting the channel into a version 2 channel.
-func (k *Keeper) AliasV1Channel(ctx context.Context, portID, channelID string) (types.Counterparty, bool) {
+func (k *Keeper) AliasV1Channel(ctx context.Context, portID, channelID string) (types.Channel, bool) {
 	channel, ok := k.channelKeeperV1.GetChannel(ctx, portID, channelID)
 	if !ok {
-		return types.Counterparty{}, false
+		return types.Channel{}, false
 	}
-	// Do not allow channel to be converted into a version 2 counterparty
+	// Do not allow channel to be converted into a version 2 channel
 	// if the channel is not OPEN or if it is ORDERED
 	if channel.State != channeltypesv1.OPEN || channel.Ordering == channeltypesv1.ORDERED {
-		return types.Counterparty{}, false
+		return types.Channel{}, false
 	}
 	connection, ok := k.connectionKeeper.GetConnection(ctx, channel.ConnectionHops[0])
 	if !ok {
-		return types.Counterparty{}, false
+		return types.Channel{}, false
 	}
 	merklePathPrefix := commitmentv2types.NewMerklePath(connection.Counterparty.Prefix.KeyPrefix, []byte(""))
 
-	counterparty := types.Counterparty{
+	channelv2 := types.Channel{
 		CounterpartyChannelId: channel.Counterparty.ChannelId,
 		ClientId:              connection.ClientId,
 		MerklePathPrefix:      merklePathPrefix,
 	}
-	return counterparty, true
+	return channelv2, true
 }
 
-// getV1Counterparty attempts to retrieve a v1 channel from the channel keeper if it exists, then converts it
+// convertV1Channel attempts to retrieve a v1 channel from the channel keeper if it exists, then converts it
```

Comment:
`convertV1Channel` makes way more sense as a name than `getV1Channel` imo given what the function does